<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>gamewindow</class>
 <widget class="QDialog" name="gamewindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1061</width>
    <height>590</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>-1</x>
     <y>0</y>
     <width>1061</width>
     <height>591</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox{
background:url(:/resources/img/main.png);
border-radius:15px;
}</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QPushButton" name="changepers_btn">
    <property name="geometry">
     <rect>
      <x>763</x>
      <y>473</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border-radius:15px;
border:0px;
background:rgb(255, 255, 255);
font-family: Geometria;
font-size: 14px;
color:black;
}
QPushButton:hover{
background: rgb(110, 87, 150);
}</string>
    </property>
    <property name="text">
     <string>  Сменить персонажа</string>
    </property>
    <property name="icon">
     <iconset resource="resource.qrc">
      <normaloff>:/resources/img/free-icon-player-substitution-3507769.png</normaloff>:/resources/img/free-icon-player-substitution-3507769.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>25</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="start_game">
    <property name="geometry">
     <rect>
      <x>983</x>
      <y>472</y>
      <width>61</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border-radius:15px;
border:0px;
background:rgb(255, 255, 255);
font-family: Geometria;
font-size: 17px;
font-weight: bold;
color:white;
}
QPushButton:hover{
background: rgb(110, 87, 150);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="resource.qrc">
      <normaloff>:/resources/img/start (2).png</normaloff>:/resources/img/start (2).png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="grp_doc">
    <property name="geometry">
     <rect>
      <x>359</x>
      <y>560</y>
      <width>381</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:rgba(0,0,0,0);
font-weight:bold;
color:rgb(166, 166, 166)</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;© 2023. Gambit Role Play. Все права защищены.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="loading_line">
    <property name="geometry">
     <rect>
      <x>33</x>
      <y>526</y>
      <width>931</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:white;
border-radius:7px;
border:3.3px solid white;
background:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(145, 102, 206, 255), stop:0.9999 rgba(145, 102, 206, 255), stop:1 rgba(255, 255, 255, 255), stop:1 rgba(255, 255, 255, 255));
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="selected_nick">
    <property name="geometry">
     <rect>
      <x>799</x>
      <y>364</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;
font-family: Geometria;
font-size: 28px;
font-weight:bold;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Bobbie Dixon&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="selected_skin">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>119</y>
      <width>201</width>
      <height>241</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:20px;
border:6px solid white;
background: url(:/resources/img/17.png) center/top;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>10</y>
      <width>755</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="forum">
           <property name="styleSheet">
            <string notr="true">color:white;
font-family: Geometria;
font-size: 16px;
color:white;
font-weihgt:bolt;</string>
           </property>
           <property name="text">
            <string>Форум</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>10</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="info">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
             <horstretch>50</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="styleSheet">
            <string notr="true">color:white;
font-family: Geometria;
font-size: 16px;
color:white;
font-weihgt:bolt;</string>
           </property>
           <property name="text">
            <string>Информация о проекте</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>10</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="rules">
           <property name="styleSheet">
            <string notr="true">color:white;
font-family: Geometria;
font-size: 16px;
color:white;
font-weihgt:bolt;</string>
           </property>
           <property name="text">
            <string>Правила форума</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_6">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>10</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="saes">
           <property name="styleSheet">
            <string notr="true">color:white;
font-family: Geometria;
font-size: 16px;
color:white;
font-weihgt:bolt;</string>
           </property>
           <property name="text">
            <string>SA ES</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>220</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="settings_btn">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
background-color:rgb(255,255,255);
background-image:url();
border-radius:15px;
}
QPushButton:hover{
background: rgb(110, 87, 150);
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="resource.qrc">
           <normaloff>:/resources/img/free-icon-settings-3524636.png</normaloff>:/resources/img/free-icon-settings-3524636.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>5</width>
           <height>2</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="close_btn">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
background-color:rgb(255,255,255);
background-image:url();
border-radius:15px;
}
QPushButton:hover{
background: rgb(110, 87, 150);
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="resource.qrc">
           <normaloff>:/resources/img/free-icon-cross-mark-17047.png</normaloff>:/resources/img/free-icon-cross-mark-17047.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>15</width>
           <height>20</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>70</y>
      <width>571</width>
      <height>361</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="styleSheet">
        <string notr="true">background:rgb(59, 54, 97) url(:/resources/img/city1.png);
opacity: 0.4;
border: 0px solid rgb(72, 62, 102);
</string>
       </property>
       <property name="title">
        <string/>
       </property>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>251</width>
          <height>41</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color:white;
font-family: Geometria;
font-size: 28px;
font-weight:bold;
border:0px;
background:url();</string>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:21pt;&quot;&gt;Новости проекта&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>50</y>
          <width>261</width>
          <height>311</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color:white;
font-family: Geometria;
font-size: 14px;
border:0px;
background:url();</string>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Пенисы!  ПЕНИСЫ! Пенис пени!&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;паапвапвапваыпвапваыпвпар&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;ывапрва&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;првапровыарпроывроапрлоыврлоапы&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;ываполывадполдыолваполдывап&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;ывфаполывдолаподлыволдапол&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;фвалпыводапдожлыводлаполдыв&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;выалпылвоаплодолдвыпаулдо&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="styleSheet">
        <string notr="true">background:rgb(59, 54, 97) url(:/resources/img/city2.png) ;</string>
       </property>
       <property name="title">
        <string/>
       </property>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>10</y>
          <width>261</width>
          <height>41</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color:white;
font-family: Geometria;
font-size: 28px;
font-weight:bold;
border:0px;
background:url();</string>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:21pt;&quot;&gt;Обновления&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>5</x>
          <y>54</y>
          <width>261</width>
          <height>311</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color:white;
font-family: Geometria;
font-size: 13px;
border:0px;
background:url();</string>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Пенисы!  ПЕНИСЫ! Пенис пени!&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;паапвапвапваыпвапваыпвпар&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;ывапрва&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;првапровыарпроывроапрлоыврлоапы&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;ываполывадполдыолваполдывап&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;ывфаполывдолаподлыволдапол&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;фвалпыводапдожлыводлаполдыв&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;выалпылвоаплодолдвыпаулдо&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
